src/main/java/com/homework/lesson15/entities/Developers.java

```diff
@@ -0,0 +1,104 @@
+package com.homework.lesson15.entities;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.extern.log4j.Log4j;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.MappedSuperclass;
+import javax.persistence.PostLoad;
+import javax.persistence.PostPersist;
+import javax.persistence.PostRemove;
+import javax.persistence.PostUpdate;
+import javax.persistence.PrePersist;
+import javax.persistence.PreRemove;
+import javax.persistence.PreUpdate;
+import javax.persistence.Table;
+import java.util.HashSet;
+import java.util.Set;
+
+@MappedSuperclass
+@Log4j
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+@Entity
+@Builder
+@Table(name = "developers")
+public class Developers {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Column(name = "name", length = 255)
+    private String name;
+
+    @Column(name = "surname", length = 255)
+    private String type;
```

Comment:
better use enum
https://vladmihalcea.com/the-best-way-to-map-an-enum-type-with-jpa-and-hibernate/